content/technotes/2018-06-12-ssh-02.md

```diff
@@ -0,0 +1,192 @@
+---
+slug: "ssh-02"
+title: "The ssh Package: Secure Shell (SSH) Client for R"
+date: 2018-06-12
+authors:
+  - name: Jeroen Ooms
+categories: technotes
+cover: "https://upload.wikimedia.org/wikipedia/commons/d/d7/8-cell.gif"
+tags:
+- R
+- packages
+- ssh
+---
+
+The new rOpenSci [ssh](https://cran.r-project.org/web/packages/ssh/index.html) package provides a native ssh client in R based on [libssh](https://www.libssh.org/), a powerful C library implementing the SSH protocol. It allows for connecting to a remote server over SSH to transfer files via SCP, setup a secure tunnel, or run a command or script on the host while streaming stdout and stderr directly to the client.
```

Comment:
Weird phrasing. It allows to connect blabla or to run? Currently the sentence is a bit hard to parse.